Rakefile

```diff
@@ -8,17 +9,11 @@ load 'tasks/compile.rake'
 load 'tasks/generate.rake'
 load 'tasks/benchmarks.rake'
 
-# TODO: remove engine check when rubinius stops crashing on RuboCop
```

Comment:
Oh nice, it stopped crashing on recent Rubinius?

Reply:
I haven't actually tested that yet (I'm about to), but Rubinius support looks have been removed some time ago (1df20eb4aa8666ceea664c01e76eac402332f039).

Reply:
Rubinius is in a bit of a rough state it seems:
https://github.com/bundler/bundler/issues/6163
https://github.com/rubinius/rubinius/issues/3769

I'll check again when either of those issues is fixed.

Reply:
Yes, true - Rubinius was in rough shape, and even as an allowed failure in Travis, was just gumming up the works to try to keep up.